src/resolve/resolved-llmnr.c

```diff
@@ -9,9 +9,16 @@
 #include "resolved-llmnr.h"
 #include "resolved-manager.h"
 
-void manager_llmnr_stop(Manager *m) {
+void manager_llmnr_stop(Manager *m, bool force) {
         assert(m);
 
+        if (!force) {
+                Link *l;
+                HASHMAP_FOREACH(l, m->links)
+                        if (link_get_llmnr_support(l) != RESOLVE_SUPPORT_NO)
+                                return;
+        }
+
```

Comment:
i don't like this pattern very much. I would have instead added a new fnction manager_llmnr_maybe_stop() that first does the loop and check and then chainloads to manager_llmnr_stop() (or doesn't).

it hink that's much nicer, and smaller probably too